docs/core-concepts/behavior.md

```diff
@@ -18,4 +18,11 @@ A special case is the `RolledBack` output. This means that the chain rolled back
 
 ## Replay of past server outputs
 
-When a `hydra-node` restarts, it will load it's history from persistence and replay previous server outputs to enable clients to re-establish their state upon re-connection. If that happens, obviously some of these outputs are not relevant anymore. One example of this is the `PeerConnected` and `PeerDisconnected`. To make it possible to determine the end of replayed history, client applications can use the `Greetings`, which will be emitted on every `hydra-node` start. See the `hydra-tui` example client for how this is handled.
+When a `hydra-node` restarts, by default it will load it's history from persistence and replay previous server outputs to enable clients to re-establish their state upon re-connection. If that happens, obviously some of these outputs are not relevant anymore. One example of this is the `PeerConnected` and `PeerDisconnected`. To make it possible to determine the end of replayed history, client applications can use the `Greetings`, which will be emitted on every `hydra-node` start. See the `hydra-tui` example client for how this is handled.
+
+Clients can optionally decide to skip history outputs and receive only the `Greetings` and following ones. In order to do that they can use query param `history=0`.
```

Comment:
I would use `no` instead of `0` to be more explicit about the fact that we don't want history.
```suggestion
Clients can optionally decide to skip history outputs and receive only the `Greetings` and following ones. In order to do that they can use query param `history=no`.
```

Reply:
or `false` or `off`?

Reply:
This seems like a good suggestion since number is not really good option here. So what do you guys prefer in the end since I am torn between `no` and `off`?

Reply:
or `false` ... what? Not helping? ;)

Reply:
Ok, used `no` in the end...